modules/tinymce/src/core/main/ts/keyboard/HomeEndKeys.ts

```diff
@@ -11,6 +11,8 @@ const executeKeydownOverride = (editor: Editor, caret: Cell<Text>, evt: Keyboard
   MatchKeys.execute([
     { keyCode: VK.END, action: MatchKeys.action(CefNavigation.moveToLineEndPoint, editor, true) },
     { keyCode: VK.HOME, action: MatchKeys.action(CefNavigation.moveToLineEndPoint, editor, false) },
+    { keyCode: VK.END, action: MatchKeys.action(CefNavigation.selectToEndPoint, editor, true), ctrlKey: true, shiftKey: true },
```

Comment:
It seems that this code path is not tested in the tests. At lest two basic tests one for each key would be useful. Also I wonder are we sure that these are not conflicting. That mac uses say Ctrl-Shift-Home for something else. Or does home even exist on a mac keyboard. Not a mac user so I have no idea. Just curious?

Reply:
I've decided to handle Cmd+Shift+Up/Down only for the mac platform and Ctrl+Shift+Home/Down for the rest.